tools/StaticAnalysis/Exceptions/AzureRM.TrafficManager/SignatureIssues.csv

```diff
@@ -18,3 +18,11 @@
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.TrafficManager\Microsoft.Azure.Commands.TrafficManager.dll","Microsoft.Azure.Commands.TrafficManager.GetAzureTrafficManagerProfile","Get-AzureRmTrafficManagerProfile","1","8510","Cmdlet 'Get-AzureRmTrafficManagerProfile' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.TrafficManager\Microsoft.Azure.Commands.TrafficManager.dll","Microsoft.Azure.Commands.TrafficManager.NewAzureTrafficManagerProfile","New-AzureRmTrafficManagerProfile","1","8100","New-AzureRmTrafficManagerProfile Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.TrafficManager\Microsoft.Azure.Commands.TrafficManager.dll","Microsoft.Azure.Commands.TrafficManager.NewAzureTrafficManagerProfile","New-AzureRmTrafficManagerProfile","1","8410","Parameter MonitorToleratedNumberOfFailures of cmdlet New-AzureRmTrafficManagerProfile does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.TrafficManager\Microsoft.Azure.Commands.TrafficManager.dll","Microsoft.Azure.Commands.TrafficManager.AddAzureTrafficManagerCustomHeaderToEndpoint","Add-AzureRmTrafficManagerCustomHeaderToEndpoint","1","8100","Add-AzureRmTrafficManagerCustomHeaderToEndpoint Does not support ShouldProcess but the cmdlet verb Add indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
All of these new cmdlets that make changes to the server should have supportshouldprocess = true.  This will allow users to pass in -Confirm which prompts them for confirmation upon run.  See here for example: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/KeyVault/Commands.KeyVault/Commands/AddAzureKeyVaultCertificateContact.cs#L101

Reply:
These are objects with two parameters that can be easily added and removed. Requiring verification would be a nuisance to the user and do more harm than good.

Reply:
The ShouldProcess loop only prompts the user if the user specifies -Confirm.  This gives the user flexibility to run cmdlets and see their possible effects before making any changes to the server.  This is a PowerShell requirement for all cmdlets that make modifications.

Reply:
Done. From what I've read it will not have any effect unless the customer uses -Confirm, so it won't get in the way.

Reply:
@allencal Looks like you enabled supportsshouldprocess correctly, but you also need to add a loop around the modification in each of the cmdlets.  See this for example of loop (line 101): https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/KeyVault/Commands.KeyVault/Commands/AddAzureKeyVaultCertificateContact.cs#L101.  Once that is done, you can remove all the these additional lines in tools/StaticAnalysis/Exceptions/AzureRM.TrafficManager/SignatureIssues.csv.